petl/comparison.py

```diff
@@ -96,9 +90,9 @@ def __getitem__(self, item):
 
 def _typestr(x):
     # attempt to preserve Python 2 name orderings
-    if isinstance(x, binary_type):
+    if isinstance(x, bytes):
```

Comment:
Wondering if the comment above is a py2 hack...

Reply:
Shall we refactor out `_typestr(x)` to `type(x).__name__`?

Reply:
- If the code is private and is not performance sensitive I wouldn't bother. 
- I didn't have the time for searching for similar cases through the changes